driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/quotactl.bpf.c

```diff
@@ -236,7 +236,7 @@ int BPF_PROG(quotactl_x,
 	if(scap_cmd == PPM_Q_GETFMT)
 	{
 		u32 quota_fmt_out_tmp = 0;
-		bpf_probe_read_user(&quota_fmt_out_tmp, sizeof(quota_fmt_out_tmp), (void *)addr_pointer);
+		bpf_probe_read_user(&quota_fmt_out_tmp, bpf_core_type_size(quota_fmt_out_tmp), (void *)addr_pointer);
```

Comment:
Hi @Andreagit97 ,

just a minor question on consistency: Above, the structure (e.g.. `user_msghdr`) is used for `bpf_core_type_size` and here the variable is used ? Not sure if we want to be consistent here as well?

Other than that, it looks good to me! Thanks!

Reply:
great catch! yes the idea was to use always the structure name in this way should be clearer when to use a core relocation or not, I will fix it thank you :)